protocol-spec/enumeration.md

```diff
@@ -127,6 +135,61 @@ None. Server-to-Client only.
 
 **Description:** Server reply to a client request for a device list.
 
+**Introduced In Version:** 0
+
+**Message Version:** 1
+
+**Fields:**
+
+* _Id_ \(unsigned int\): Message Id
+* _Devices_ \(array\): Array of device objects
+  * _DeviceName_ \(string\): Descriptive name of the device
+  * _DeviceIndex_ \(unsigned integer\): Index used to identify the device when sending Device Messages.
+  * _DeviceMessages_ \(dictionary\): Accepted Device Messages 
+    * Keys \(string\): Type names of Device Messages that the device will accept
+    * Values \(dictionary\): Attributes for the Device Messages. Both keys and values are strings.
+
+**Expected Response:**
+
+None. Server-to-Client message only.
+
+**Flow Diagram:**
+
+![img](devicelist_diagram.svg)
+
+**Serialization Example:**
+
+```json
```

Comment:
If we're going to add generic messages with constraints in this version, we should use those as an example here. Something like VibrateCmd with number of motors.

Reply:
This has been addressed